docs/cubeviz/displaycubes.rst

```diff
@@ -45,18 +45,50 @@ visible (checked) or invisible (unchecked).
 Changing Wavelength Slice
 =========================
 
-To change the wavelength slice, click and drag the :guilabel:`slider` at the top of the Cubeviz UI
+To change the wavelength slice, click and drag the :guilabel:`slider` at the top of the Cubeviz User Interface (UI)
 to the left or right in order to get a lower or higher slice number, respectively. Another option is to
 click the :guilabel:`circle` that controls the location on the :guilabel:`slider` and use the
 :guilabel:`left` or :guilabel:`right` arrow keys on your keyboard to find a slice number with more precision.
 A third option is to type a slice number into the :guilabel:`text box` to the right of the :guilabel:`slider`
 and press :guilabel:`enter` on your keyboard. A fourth option is to use the :guilabel:`up/down arrows` inside
 the :guilabel:`text box` to raise/lower the slice number.
 
-Defining and Selecting Region Subsets
-=====================================
+Defining Spatial Regions
+========================
+
+Regions can be created by first clicking the :guilabel:`hammer and screwdriver` icon and
+selecting either the :guilabel:`rectangular region of interest` icon or the
+:guilabel:`circular region of interest` icon, then clicking and dragging (using the mouse) until the
+desired region is covered by a colored shape, after which you release the mouse button.
+Once you have completed the action of clicking and dragging, there is an opportunity to reposition
+the region of interest by clicking on it and moving the region to a different location in the image viewer.
+If you are satisfied with the positioning of your region of interest, simply click another part of the
+viewer to lock it in place. Once a region is selected, the cube will be collapsed in wavelength space over the region,
```

Comment:
> simply click another part of the viewer to lock it in place.

Oh really? You have to actually click somewhere else in order for the moved region to stay? I feel like the user should just be able to let go and it should just work? I'd suggest we should change this behavior later

Reply:
Yes, I think the idea is that you should be able to move the region around before locking it in place, so it does not lock until you click elsewhere in the image viewer.

Reply:
The existing behavior allows you to continue editing a region with the selected logical function until you click elsewhere. 
Maybe don't require the click and the region remains editable until another subset is generated?  However, that would
mean autogenerating the collapse after each sub-region edit.